### PR TITLE
Update handling for setting contentful content links to have target _…

### PR DESCRIPTION
**Pull request by mcstover**
…blank

### CODE REVIEW

File: src/components/CorporateCampaign/CampaignPartner.vue [hunk 1]
```diff
@@ -70,7 +70,11 @@ export default {
 		// make sure all partner content links open externally
 		if (this.$refs.partnerBodyCopy) {
 			const links = this.$refs.partnerBodyCopy.querySelectorAll('a');
-			links.forEach(link => { link.target = '_blank'; }); // eslint-disable-line no-param-reassign
```
Comment (ryan-ludwig) [inline src/components/CorporateCampaign/CampaignPartner.vue:73]: Haven't tested in IE, but I think Array.from would do the trick too
`Array.from(links).forEach(link => { link.target = '_blank'; }); // eslint-disable-line no-param-reassign)`

Comment (ryan-ludwig) [reply src/components/CorporateCampaign/CampaignPartner.vue:73]: Oh, nevermind. https://gomakethings.com/converting-a-nodelist-to-an-array-with-vanilla-javascript/. Unless maybe babel covers that by default.

**Review by ryan-ludwig — APPROVED** [review]

File: src/components/CorporateCampaign/CampaignPartner.vue (hunk 1 above)
Comment (mcstover) [reply src/components/CorporateCampaign/CampaignPartner.vue:73]: yep big ol 🤦  on my part. @emuvente helped me sort that out.

**Review by emuvente — APPROVED** [review]
👍

**Pull request merged**